web/siteplan/src/model/SvgElement.ts

```diff
@@ -91,7 +91,7 @@ export class SvgBridgeSignal extends SvgElement {
     nullpunkt: ISvgPoint | null,
     mountOffset: number,
     mountDirection: MountDirection,
-    signalLabel: Label | null,
+    signalLabel: Label | undefined,
```

Comment:
```suggestion
    signalLabel?: Label,
```
for consistent with another label attribute